packages/cozy-scanner/src/DocumentQualification.spec.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+jest.mock('cozy-ui/transpiled/react/utils/color', () => ({
+  getCssVariableValue: () => '#fff'
+}))
+import { DocumentQualification } from './DocumentQualification'
+import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
+
+const mockDate = () => {
+  const DATE_TO_USE = new Date('2019')
+  const _Date = Date
+  global.Date = jest.fn(() => DATE_TO_USE)
+  global.Date.UTC = _Date.UTC
+  global.Date.parse = _Date.parse
+  global.Date.now = _Date.now
+}
+
+const unMockDate = () => {
+  global.Date = new Date()
+  global.Date.UTC = Date.UTC
+  global.Date.parse = Date.parse
+  global.Date.now = Date.now
+}
```

Comment:
Why not use [`mockdate`](https://github.com/konnectors/toutatice/blob/d983393ff8132579a6849f8996a2a8d2cf7a8982/src/helpers/transpileGroupToCozy.spec.js#L8-L14) instead?

Reply:
Thanks